src/lib.rs

```diff
@@ -570,7 +570,7 @@ impl<C: Context> Secp256k1<C> {
     }
 
     /// Uses the ffi `secp256k1_context_preallocated_size` to check the memory size needed for a context
-    pub fn preallocate_size() -> usize {
+    pub(crate) fn preallocate_size_gen() -> usize {
```

Comment:
I think we should still export this. It's needed to estimate the allocation size in generic contexts.

Reply:
I mostly didn't want to confuse the user with too many functions, some generic and some not